JOURNAL.md

```diff
@@ -3,13 +3,21 @@ A place to keep track of all our mentees, mentors, and mentorship rounds.
 
 Click here for the [List of Mentors][] :heart:.
 
+## Second Round
+Duration of the second round is 10 weeks.
+
+| Mentor                       | Mentee                             | Scope             | Start Date   | Diary      | Graduation |
+|------------------------------|------------------------------------|-------------------|--------------|------------|------------|
+
 ## First Round
+The duration of the first round is 1 month.
+
 | Mentor                       | Mentee                             | Scope             | Start Date   | Diary      | Graduation |
 |------------------------------|------------------------------------|-------------------|--------------|------------|------------|
-| [Dan Shaw (@dshaw)][]        | [Ahmad Bamieh (@Bamieh)][]         | [CommComm][]      | 12 Jul, 2018 | [#79](https://github.com/nodejs/mentorship/issues/79) ||
-| [Stephen Belanger (@Qard)][] | [Princiya Sequeira (@princiya)][]  | [Diagnostics][]   | 5 Aug, 2018  | [#85](https://github.com/nodejs/mentorship/issues/85) ||
-| [Benjamin Coe (@bcoe)][]     | [Jennifer Bland (@ratracegrad)][]  | [Testing][]       | 6 Aug, 2018  | [#87](https://github.com/nodejs/mentorship/issues/87) ||
-| [Gus Caplan (@devsnek)][]    | [Kostas Kapetanakis (@kapekost)][] | [VM][]            | 24 Aug, 2018 | [#90](https://github.com/nodejs/mentorship/issues/90) ||
+| [Dan Shaw (@dshaw)][]        | [Ahmad Bamieh (@Bamieh)][]         | [CommComm][]      | 12 Jul, 2018 | [#79](https://github.com/nodejs/mentorship/issues/79) |:+1:|
+| [Stephen Belanger (@Qard)][] | [Princiya Sequeira (@princiya)][]  | [Diagnostics][]   | 5 Aug, 2018  | [#85](https://github.com/nodejs/mentorship/issues/85) |:+1:|
+| [Benjamin Coe (@bcoe)][]     | [Jennifer Bland (@ratracegrad)][]  | [Testing][]       | 6 Aug, 2018  | [#87](https://github.com/nodejs/mentorship/issues/87) |:+1:|
+| [Gus Caplan (@devsnek)][]    | [Kostas Kapetanakis (@kapekost)][] | [VM][]            | 24 Aug, 2018 | [#90](https://github.com/nodejs/mentorship/issues/90) |:-1:|
```

Comment:
```suggestion
| [Gus Caplan (@devsnek)][]    | [Kostas Kapetanakis (@kapekost)][] | [VM][]            | 24 Aug, 2018 | [#90](https://github.com/nodejs/mentorship/issues/90) ||
```

Let's not use :-1:. Life happens. I suggest we either leave it blank for those who did not graduate or find something that conveys a more neutral or positive reaction.